internal/lookout/db-gen/db_gen_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"database/sql"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/armadaproject/armada/internal/lookout/repository/schema"
```

Comment:
This formatting looks off.  Can you indent import and remove a line?

Reply:
The linting check failed in the CI before. I ran `golangci-lint run --fix` and it introduced this new line.

Reply:
The linter separates local imports from third party imports I believe. like [here](https://github.com/armadaproject/armada/blob/5e5f55c65f0e99faf3b8d319de849c480652ca56/internal/armadactl/queue.go#L9)

Reply:
It looks good now. It was the indenting but maybe it was a big in UI